sentinel-core/src/test/java/com/alibaba/csp/sentinel/util/IdUtilTest.java

```diff
@@ -0,0 +1,12 @@
+package com.alibaba.csp.sentinel.util;
```

Comment:
Could you please add the license header here?

Reply:
@sczyh30 done.